react-frontend/src/components/StyleComponents/bannerWithImage.js

```diff
@@ -80,6 +80,53 @@ export const BannerTitle = styled.h1.attrs({
   }
 `;
 
+export const BannerSideImageImg = styled.img.attrs({
+  className: 'sideImage',
+})`
+  background: transparent;
+  background-color: #f2f2f2;
+  display: block;
+  max-width: 100%;
+  max-height: 100%;
+  padding-bottom: 20px;
+  padding-top: 20px;
+`;
+
+export const BannerSideImgText = styled.div.attrs({
+  className: 'sideImageText',
+})`
+  padding-bottom: 20px;
+  padding-top: 20px;
+`;
+
+export const BannerSideImgTitle = styled.h1.attrs({
+  className: 'bannerTitle',
+})`
+  color: #313132;
+  font-family: ‘BC Sans’, ‘Noto Sans’, Verdana, Arial, sans-serif;
+  font-size: 37px;
+  font-weight: bold;
+  line-height: 1.2;
+  margin-bottom: 10px;
+  text-align: left;
+  @media only screen and (max-width: 800px) {
```

Comment:
I realized we should have used max-width: 767px for this breakpoint and the one below—so that the text alignment switches right at the same time the columns break. But this isn't a big deal.